File: DSA/SakshamDwivedi_590013147/Day16/D16P2.py
#Solution Function
def occurrence(arr, target):
    count = 0
    for i in range(len(arr)):
        if arr[i] == target:
            count += 1
    return count

#Fixed Input
array = [1, 2, 3, 2, 2, 4]
tar = 2
print(occurrence(array, tar))
    
#Custom Input
array = list(float(x) for x in input("Enter the array : ").split())
tar = input("Enter the target value : ")
if tar == "":
    tar = None
else:
    tar = float(tar)
print(occurrence(array, tar))

File: DSA/SakshamDwivedi_590013147/Day16/D16P2.c
#include <stdio.h>

float arr[100];
float target;
int size;
int count = 0;

int main(){
    printf("Enter the size of the array : ");
    scanf("%d", &size);
    printf("Enter the elements : ");
    for(int i = 0; i < size; i++){
        scanf("%f", &arr[i]);
    }
    printf("Enter the target value : ");
    scanf("%f", &target);

    for(int i = 0; i < size; i++){
        if (arr[i] == target){
            count++;
        }
    }
    printf("%d", count);
}
